test/integration/blockchain-events.spec.ts

```diff
@@ -14,14 +14,16 @@ const expect = chai.expect
 async function createAgreement (app: TestingApp, file: File, billingPeriod: number, money: number, size?: number): Promise<string> {
   const encodedFileHash = encodeHash(file.fileHash)
 
+  const agreementSizeRounded = Math.floor(size ?? file.size) + 1
+  const agreementSize = agreementSizeRounded >= 1 ? agreementSizeRounded : 1
```

Comment:
What are you trying to achieve here? 🤔 If I understand this correctly you are trying to guarantee that size is at least 1? Maybe `Math.max(1, size ?? file.size)` could be better solution?

Reply:
also, I try to guarantee that if we have file.size = 2.32 then the agreement size will be rounded to 2 and then we have size error

Reply:
Ok then `Math.max(1, Math.ceil(size ?? file.size))`?

Reply:
Actually I think `Math.ceil(size ?? file.size)` will be always at least 1 no?

Reply:
Yes

Reply:
Fixed